network.go

```diff
@@ -1,150 +1,153 @@
-package p2p
-
-import (
-	"math/rand"
-	"time"
-
-	log "github.com/sirupsen/logrus"

```

Comment:
Logrus is pretty heavy as a logging package.  I see that the live feed uses it too, but I am not entirely sure its value justifies its overhead. 

Here is the [results](http://hackemist.com/logbench/) from one set of [benchmarks](https://github.com/rs/logbench)

Reply:
I picked logrus because that's one of the logging packages used throughout factomd and the goal was to keep is as consistent as I could to aid integration and transfer experience of people familiar with factomd.

The benchmarks are pretty bad, though. Are there any plans to replace factomd's logrus with something else? Would switching out the logging package still be compatible with factomd's existing infrastructure / elk stash?